### ISSUE TITLE
Filter games per URL-based vs locally stored background image

### ISSUE THREAD

**Issue by bettafo** [JosefNemec/Playnite#2236]
**Check for existing issue**
No similar feature request exists as far as I could see.

**Describe the solution you'd like**
Selecting games or displaying details about games which backgound image path was defined as an URL adress (as opposed to a locally stored image in the library) can be prone to issues: When the Internet connection has problems or the website that hosts the image is temporarily difficult/slow to access or when the URL has been taken down then Playnite can slow down significantly when selecting the game (desktop view) or even freeze for the pretty long moment.

It would be nice to be able to filter games that fetch their background image from an URL (rather than from the locally stored library) to be able to manually download and locally store the image instead.

**Comment by JosefNemec** [comment]
I don't think this is something that should be available as a standard filter, because it's a really niche filter option and filtering panel is already overcrowded. This is something that should be IMHO done as a custom extension.

I will leave this open, but I don't think it's something that will be implemented in near future.

**Comment by bettafo** [comment]
I understand, and I agree that this kind of feature should not interfere with the general-use filter. Rather it belongs to a maintenance logic and should herefore enter Settings like other maintenance related features have already (Library  Manage, etc.). There are other maintenance-related searches already requested (games paths etc) and they could be the object of a section in Settings. Or this could be grouped with other media-related settings when they get implemented (like the ones about selecting media sizes to be downloaded or re-sizing media etc.).
